packages/nest/src/generators/resource/resource.ts

```diff
@@ -32,6 +32,7 @@ function normalizeResourceOptions(
   options: ResourceGeneratorOptions
 ): NormalizedOptions {
   return {
+    ...options,
```

Comment:
Thank you, can you please make this change in the util instead so that all of the generators are fixed?

Reply:
Good suggestion, I have made the changes. Thank you!